### DISCUSSION TITLE
Test 5 2 6 Router Downgrade Threshold REED Failed

### DISCUSSION THREAD

**Discussion by xiaojiang2017** [openthread/openthread#9052] [Q&A]
**Describe the bug** A clear and concise description of what the bug is.

Our OTBR is in the process of Thread 1.3 certification. But Failed in Test case 5.2.6.
We use GSDK v4.2.3.
We found the related issues https://threadgroup.atlassian.net/browse/TESTPLAN-523.
How to solve this problem? Looking forward to your reply. Thanks.
[Fail_5_2_6_Router_v58.0.zip](https://github.com/openthread/openthread/files/11469224/Fail_5_2_6_Router_v58.0.zip)

**Comment by jwhui** [comment]
As noted in [TESTPLAN-523](https://threadgroup.atlassian.net/browse/TESTPLAN-523), there are situations when a Thread 1.3 Border Router is not required to downgrade to a child. Please reach out to the Thread Group Certification Director if you have further questions.

**Reply by xiaojiang2017** [reply]
Hi @jwhui 
The Thread Group reply is as follow,
![image015(05-16-11-30-48)](https://github.com/openthread/openthread/assets/73470725/659a0fe1-7b4e-4918-b123-a28ff2c4830a)

I found 3 related PRs in SPEC-1033. 
PR: [[netdata] get the list or count of RLOC16s of border routers by abtink · Pull Request #7543 · openthread/openthread](https://github.com/openthread/openthread/pull/7543) 
PR: [[mle-router] leader to handle "BR request" in an Address Solicit by abtink · Pull Request #7578 · openthread/openthread](https://github.com/openthread/openthread/pull/7578) 
PR: [[netdata] add feature for BR to request router role upgrade by abtink · Pull Request #7597 · openthread/openthread](https://github.com/openthread/openthread/pull/7597)
But a lot of changes have been made in GSDK v4.2.3 and the latest version. The ALT also failed the test using the Raspberry Pi. Did the update bring new problems?

**Reply by jwhui** [reply]
The functionality for BR opting not to downgrade to child was added in:
- https://github.com/openthread/openthread/pull/7597

While the code has changed since, the same functionality remains:
https://github.com/openthread/openthread/blob/7c5dca178fa68968e0e2a13f6d164645c654361d/src/core/thread/mle_router.cpp#L3847-L3850
